composer.json

```diff
@@ -57,4 +57,4 @@
     "format-check": "php vendor/bin/php-cs-fixer fix -v --dry-run --using-cache=no .",
     "test": "php vendor/bin/phpunit tests"
   }
-}
+}
```

Comment:
I believe we should have a newline at the end of this file

Reply:
The VS Code formatter got rid of the newline, but added it via GH editor now.